elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -17,7 +19,7 @@ import './bulbs-slick-slideshow.scss';
 /// 7. On l/r keydown, trigger slidechange.
 class BulbsSlickSlideshow extends BulbsHTMLElement {
   attachedCallback () {
-    this.slideshow = $('.slider');
+    this.slideshow = $('.slider').last();
```

Comment:
was never an issue in the browser, which is the weird part. but this still works, just somewhat slower since it queries the entire page.